config/topological_inventory.yml

```diff
@@ -0,0 +1,15 @@
+stable:
```

Comment:
While the code says "branch", that's a poor naming choice on my side.  Think of it more as a "repo_set".  So in that way you need a unique name.  In this case, this stable will collide with the sources stable.

I think here, you could use `topological-stable` perhaps (and `sources-stable`)